src/core/api/health.go

```diff
@@ -77,6 +79,9 @@ func (h *HealthAPI) CheckHealth() {
 		}
 		components = append(components, componentStatus)
 	}
+
+	sort.Slice(components, func(i, j int) bool { return components[i].Name < components[j].Name })
```

Comment:
why sort components here? I think the order is doesn't matter.

Reply:
Keep the components in order will make the response more readable when reading by the human.

And the stable response content is required when we want to enable cache-control ([ETag](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/ETag)) for the HTTP request. 

Of cause, the harbor doesn't enable the ETag currently. I sort the components only want to make it more readable when I check the health using curl.